db/migrate/20230822190043_add_clean_case_number_to_court_cases.rb

```diff
@@ -0,0 +1,35 @@
+class AddCleanCaseNumberToCourtCases < ActiveRecord::Migration[7.0]
```

Comment:
I really don't think that we need clean case number on the `court_cases` table. If you can find some examples, I'm open to it, but in my mind the clean case number column is getting the case number into the OSCN format